Makefile

```diff
@@ -68,6 +67,8 @@ docker-build:
 	cd scripts; \
 	docker build $(BUILD_PROXY) -t $(IMAGE_BUILDER) . 
 
+clean:
+	rm -rf "$(WORKDIR)"
```

Comment:
I would prefer to remove "rm -rf "$(WORKDIR)/rootfs" and "rm -rf "$(WORKDIR)/linux" to avoid remove something more dangerous.

Reply:
and "$(WORKDIR)/img" ?

Reply:
yeah